ui/components/app/connected-accounts-permissions/connected-accounts-permissions.component.js

```diff
@@ -64,20 +64,25 @@ export default class ConnectedAccountsPermissions extends PureComponent {
         >
           <p>{t('authorizedPermissions')}:</p>
           <ul className="connected-accounts-permissions__list">
-            {permissions.map(({ key: permissionName }) => (
-              <li
-                key={permissionName}
-                className="connected-accounts-permissions__list-item"
-              >
-                <CheckBox
-                  checked
-                  disabled
-                  id={permissionName}
-                  className="connected-accounts-permissions__checkbox"
-                />
-                <label htmlFor={permissionName}>{t(permissionName)}</label>
-              </li>
-            ))}
+            {permissions.map(({ key }) => {
+              const permissionName =
+                key === 'eth_accounts' ? 'permission_ethereumAccounts' : key;
```

Comment:
It looks like this only fixes the localized message for the `eth_accounts` permission. All of the other permissions would result in the same crash, because none of their descriptions match the permission name either.

Reply:
After reading this and looking over more code, I realized that there was a system in `ui/components/app/permissions-connect-permission-list/permissions-connect-permission-list.js` that was retrieving permissions labels. The two files are now sharing the logic to retrieve labels. We preserved the `key`/`permissionName` logic.